.gitignore

```diff
@@ -1,2 +1,3 @@
 /lib/
 /node_modules/
+package-lock.json
```

Comment:
Missing newline at end of file.

Reply:
Updated all of the changes in the new commit.